packages/cypress/src/index.ts

```diff
@@ -127,8 +127,16 @@ export const onBeforeBrowserLaunch = (
   // we don't use the browser parameter but we're keeping it here in case we'd ever need to read from it
   // (this way users wouldn't have to change their cypress.config file as it's already passed to us)
   browser: Cypress.Browser,
-  launchOptions: Cypress.BeforeBrowserLaunchOptions
+  launchOptions: Cypress.BeforeBrowserLaunchOptions,
+  config: Cypress.PluginConfigOptions
 ) => {
+  // when Cypress is in interactive mode, we won't be snapshotting.
```

Comment:
Why won't we be snapshotting?

Reply:
@tevanoff we won't be snapshotting because this is the "manual mode" of Cypress (where no tests are run unless the user clicks through the spec files in the sidebar, similar to `yarn playwright test --ui`).

Reply:
Hm ok, although we do run all the archiving and snapshotting in playwright and it has been helpful when debugging issues. Is it just disabled because the `ELECTRON_EXTRA_LAUNCH_ARGS` would be necessary to run in the ui mode?

Reply:
@tevanoff the main reason I disabled it is so that users don't have to prepend their `cypress open` command with our `ELECTRON_EXTRA_LAUNCH_ARGS` env variable (I figure they're probably already not thrilled about having to add env variable to their `cypress run` command). Also, this env variable is only necessary if they end up running their tests in Electron (that's the default browser for `cypress run`, but for `cypress open` it gives you a browser picker to choose).

The other reason is that Cypress' Interactive mode doesn't fire all of the lifecycle hooks when it runs. For example, it doesn't clear out the `cypress/downloads` directory when running `cypress open` like it does for `cypress run`, thus preventing a clean archiving from test run to test run. That's less of a dealbreaker but also seemed to be a potential problem for people.

Happy to chat if that'd help.

Reply:
@skitterm that all makes sense, thanks!